src/components/Form/SetAclPermissionForm.jsx

```diff
@@ -109,51 +76,80 @@ const SetAclPermissionForm = () => {
 
   return (
     <FormSection
-      title="Set Permission to File"
+      title="Permission for Document"
       state={state}
-      statusType="Permission status"
-      defaultMessage="To be set..."
+      statusType="Status"
+      defaultMessage="No action yet..."
     >
       <Box display="flex" justifyContent="center">
         <form onSubmit={handleAclPermission} autoComplete="off">
-          <FormControl fullWidth>
-            <InputLabel htmlFor="set-acl-to" label="Set permissions to:" />
+          <Typography
+            variant="subtitle2"
+            mb={2}
+            sx={{
+              width: '250px',
+              whiteSpace: 'nowrap',
+              overflow: 'hidden',
+              textOverflow: 'ellipsis'
+            }}
+          >
+            <span style={{ color: 'grey' }}>File Name: </span>
+            LongAssFileNameToTestIfThisIsWorkingCorrectly
```

Comment:
Think we can remove this test line if the truncation is working properly.

Reply:
Don't want a placeholder with comment to know where to put it when we have the context?

Reply:
A TODO comment would be fine.